app/templates/books/employee.html

```diff
@@ -0,0 +1,19 @@
+{% extends "books/bookbase.html" %}
+{% import "bootstrap/wtf.html" as wtf %}
+
+{% block title %}Pracownik{% endblock %}
+
+{% block page_content %}
+<h1>Witaj Nazwa Pracownika</h1>
+{% endblock %}
+
+{% block navbar %}
+    {{ super() }}
+{% endblock %}
+  {% block navbar_content %}
+    <li><a href="/Books">Books</a></li>
```

Comment:
W pozostały miejscach wydaje się, że interfejs jest po polsku. W takim razie Navbar też powinien być
```suggestion
    <li><a href="/Books">Książki</a></li>
```